jsonschema/current/Interfaces.schema.json

```diff
@@ -97,6 +93,15 @@
             "oneOf": [
                 {"$ref": "dungeoneer/statuseffects/StatusEffect.schema.json" }
             ]
+        },
+        "TriggerLike": {
+            "oneOf": [
+                {"$ref": "dungeoneer/entities/triggers/ButtonModel.schema.json"},
```

Comment:
I'm confused a bit. `ButtonModel` does not inherit from `Trigger`. So e.g. `Actor::useTrigger` would be wrong.

Reply:
Thanks for catching this. Fixed!